editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -124,11 +110,62 @@ function Component(props: Props) {
     }
   }, [data, address]);
 
-  useEffect(() => {
-    if (geojson) setBoundary(geojson);
-  }, [geojson]);
+  const validateAndSubmit = () => {
```

Comment:
Refactoring to `validateAndSubmit` & `getBody` are consistent with DrawBoundary, which should make managing these components feel a bit more aligned in the future